src/main/java/org/apache/tomcat/jakartaee/ManifestConverter.java

```diff
@@ -112,7 +107,7 @@ private boolean updateValues(Attributes attributes, EESpecProfile profile) {
         if (attributes.containsKey(Attributes.Name.IMPLEMENTATION_VERSION)) {
             String newValue = attributes.get(Attributes.Name.IMPLEMENTATION_VERSION) + "-" + Info.getVersion();
             attributes.put(Attributes.Name.IMPLEMENTATION_VERSION, newValue);
-            result = true;
+            // Purposefully avoid setting result
```

Comment:
This is metadata and would cause every file to be considered converted, even if no interesting conversions took place. The destination manifest is written regardless with this change. Omitting this is what allows `hasConverted()` to work.